backend/src/test/java/resources/application.properties

```diff
@@ -1,3 +1,3 @@
 server.port=${PORT:8080}
-spring.data.mongodb.uri=${mongodb://localhost/OrrAdbTest?retryWrites=true&w=majority}
-secret=${test123}
\ No newline at end of file
+spring.data.mongodb.uri=mongodb://localhost/OrrAdbTest?retryWrites=true&w=majority
+secret=test123
```

Comment:
Now I wonder what kind of secret this is :) Maybe add a namespace like `jwt.secret` or `github.client.secret` or whatever it is. Also if this is really a secret you don't want to commit it but read it from an enironment variable like you did with server.port

Reply:
It's only for the Tests. The real srecret is hidden in an Environment-variable.  😉

Reply:
doh 🤦 test folder.. never mind